### PR TITLE
{lib}[GCCcore/11.2.0] libpsl v0.21.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+dependencies = [
+    ('libidn2', '2.3.2'),
+    ('libunistring', '1.0'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/psl', 'lib/libpsl.a'],
+    'dirs': []
+}
+
```
Comment (Micket) [inline easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb:31]: sanity_check_commands = [('psl -h')]

File: easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+source_urls = ['https://github.com/rockdaboot/libpsl/archive']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ac6ce1e1fbd4d0254c4ddb9d37f1fa99dec83619c1253328155206b896210d4c']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('libtool', '2.4.6'),
+    ('Autotools', '20210726'),
+    ('CMake', '3.22.1'),
```
Comment (Micket) [inline easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb:19]: So.. this stuck out as odd to me. There is no cmake here at all. And, why libtool and Autotools? It's not generating any configure script..  I tried to look it up and i can't see any reason why any of these were added. Removing all 3 made no difference in my test. I see i bumped this last time and probably just overlooked it, and before that  it was added without any explanation.

lets get rid of it. I do however that Python was dropped as a dep.. but this still looks like it ought to be a builddep. 

```suggestion
    ('Python', '3.9.6'),
```

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb:31]: Good idea, thanks!

File: easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb (hunk 2 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/l/libpsl/libpsl-0.21.1-GCCcore-11.2.0.eb:19]: Sounds good to me. As that was an update from an existing EC file, I just took it as it is. I have added `Python` but I added it as the `-bare` one, as it is only required as a build-dependency. Maybe that is the reason why that was not discovered before.

**Comment by akesandgren** [comment]
@Micket ok with the change?

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.5, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/df0ff0ebe88dfb0584d280fb268c4b9d for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node3127.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/7782ace76ae96f2ad13317cca91f849e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**